src/renderer/features/contacts/CreateContact/model/contact-form.ts

```diff
@@ -0,0 +1,97 @@
+import { attach, createApi, createStore, forward, sample } from 'effector';
+import { createForm } from 'effector-forms';
+
+import { Contact, contactModel } from '@renderer/entities/contact';
+import { toAccountId, validateAddress } from '@renderer/shared/lib/utils';
+import { validateFullUserName } from '@renderer/shared/api/matrix';
+
+export type FormApi = {
+  onSubmit: () => void;
+};
+const $formApi = createStore<FormApi | null>(null);
+const formApi = createApi($formApi, {
+  apiChanged: (state, props: FormApi) => ({ ...state, ...props }),
```

Comment:
what is the reason to call it in that way? It looks little bit strange and I didn't find any such naming, maybe you can provide examples?  why not `formChanged`?